Formula/docker-machine-driver-xhyve.rb

```diff
@@ -23,6 +23,8 @@ class DockerMachineDriverXhyve < Formula
     depends_on "libev"
   end
 
+  patch :DATA
```

Comment:
That's patch is not merge to upstream yet. Because `$(LIBEV_FILE)` is just carried from
https://github.com/docker/hyperkit/blob/master/Makefile#L83
I would like to follow hyperkit as much as possible. but, Now I'm considering allowing the user to specify the path. So, there is no commit url yet.

Also, sometimes Go users set `$GO_LDFLAGS` to shell global. i.e. always strip symbol table & DWARF with `-s -w`.
In this case, if change the `+=` to `?=`, GNU Make handling shell `$GO_LDFLAGS` env(IIRC).
It means, ignores `-X=xhyve.GitCommit=(...)` trick.
There is a possibility that I can not take a commit hash on the issue, So I am not going to change upstream.

Well, If some comment is necessary, I would appreciate it if I could get your advice, because I am not good at English...